### ISSUE TITLE
Graded lab: Writing methods

### ISSUE THREAD

**Issue by hammad-ibrahim** [DevOps-With-Hammad/Java-Amazon_software_development#1]
**As a** [role]  
 **I need** [To be good at making Methods in Java in the right way and syntax]  
 **So that** [I can master the basics of the Java as programming language ]  
   
 ### Details and Assumptions
 * [-I have mastered the first steps of the fundamental of  Java also the basics 
 * - also the step after that of how to take input and making process and giving output based on that ]
   
 ### Acceptance Criteria  
   
 ```gherkin
 Given [some context]
 When [certain action is taken]
 Then [the outcome of action is observed]
 ```

**Comment by satyyam11** [comment]
can i help?

**Comment by hammad-ibrahim** [comment]
> can i help?

Yes for sure . I am just started my learning , so I am putting all the efforts in fundamentals and basics  so that I can be good at the OOB and understand it faster you can help and participate here as much as you like . Thanks my Dear

**Comment by satyyam11** [comment]
How can I be of any help

**Comment by hammad-ibrahim** [comment]
You can fork this project and create a new repo with a simple UI pages one or tow at least with header and footer with any content you like and the body you can put in a simple login or any thing that take date and store it in a database -- I wish this project be with JAVA BOOT String and any database you like with documentation please . Thanks

**Comment by satyyam11** [comment]
I cant do frontend tell me if i can help with backend something
